From 583462bede866e3111f7a18de1e9b8c7fe6f6986 Mon Sep 17 00:00:00 2001
From: "Tekriwal, Tanuj" <tanuj.tekriwal@intel.com>
Date: Thu, 2 Jan 2025 07:51:37 +0000
Subject: [PATCH] Using private libs instead of google libs.

Renaming the libs to libxx_orig

Tests Done: Build and boot

Tracked-On: OAM-128386
Signed-off-by: Tekriwal, Tanuj <tanuj.tekriwal@intel.com>
---
 tests/modetest/Android.bp | 2 +-
 tests/proptest/Android.bp | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/modetest/Android.bp b/tests/modetest/Android.bp
index 995ce776..0b38f094 100644
--- a/tests/modetest/Android.bp
+++ b/tests/modetest/Android.bp
@@ -4,7 +4,7 @@ package {
     // all of the 'license_kinds' from "external_libdrm_license"
     // to get the below license kinds:
     //   SPDX-license-identifier-MIT
-    default_applicable_licenses: ["external_libdrm_license"],
+    default_applicable_licenses: ["external_libdrm_license_orig"],
 }
 
 build = ["Android.sources.bp"]
diff --git a/tests/proptest/Android.bp b/tests/proptest/Android.bp
index f9051d79..696d971e 100644
--- a/tests/proptest/Android.bp
+++ b/tests/proptest/Android.bp
@@ -4,7 +4,7 @@ package {
     // all of the 'license_kinds' from "external_libdrm_license"
     // to get the below license kinds:
     //   SPDX-license-identifier-MIT
-    default_applicable_licenses: ["external_libdrm_license"],
+    default_applicable_licenses: ["external_libdrm_license_orig"],
 }
 
 cc_binary {
-- 
2.34.1

